### DISCUSSION TITLE
Open LCP-protected ePUB in Testing app(Kotlin-toolkit)

### DISCUSSION THREAD

**Discussion by EdKweon** [readium/kotlin-toolkit#527] [Q&A]
Hello,

As a junior developer, I am trying to add functionality for viewing LCP-protected documents in my reading app. I attempted to view LCP-protected documents using the app from the Kotlin-toolkit, but it says that LCP-protected documents were not compiled. Is there an additional module I need to download? Or is the testApp not developed to support LCP-protected documents?

Thank you.

**Comment by mickael-menu** [comment]
Take a look at this guide which works with the latest `3.0.0-beta.1`: https://github.com/readium/kotlin-toolkit/blob/develop/docs/guides/lcp.md

The Test App can open LCP publications but you need to setup the private "test" liblcp provided by EDRLab. You should contact them to get the instructions.

**Reply by EdKweon** [reply]
Thank you for your reply.
